.github/workflows/install.yml

```diff
@@ -0,0 +1,64 @@
+name: Install
+
+on:
+  push:
+  workflow_dispatch:
+
+jobs:
+  install-matrix:
+    strategy:
+      matrix:
+        os: [macos-latest, windows-latest, ubuntu-latest]
+        build_shared_libs: [0, 1]
+        system_curl_uuid: ["False"]
+        external_sqlite: [1]
+        include:
+        # Extra Linux jobs for using system curl/libuuid
+          - os: ubuntu-latest
+            build_shared_libs: 0
+            system_curl_uuid: "True"
+            external_sqlite: 1
+          - os: ubuntu-latest
+            build_shared_libs: 1
+            system_curl_uuid: "True"
+            external_sqlite: 1
+        # Extra jobs to test installer with SQLite3 compiled into Snowplow
+          - os: macos-latest
+            build_shared_libs: 1
+            system_curl_uuid: "False"
+            external_sqlite: 0
+          - os: windows-latest
+            build_shared_libs: 1
+            system_curl_uuid: "False"
+            external_sqlite: 0
+          - os: ubuntu-latest
+            build_shared_libs: 1
+            system_curl_uuid: "False"
+            external_sqlite: 0
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Install Conan
+        id: conan
+        uses: turtlebrowser/get-conan@main
+        with:
+          version: 1.60.0
```

Comment:
Is there any specific reason why you didn't make use of Conan version 2? I understand this is only internal usage, but would be great to have the dependency up to date if possible.

Reply:
I had this experience that many recipes didn't work with Conan 2. But that was right after the release. I'll try to upgrade to V2, the depencies here are quite common, should work with V2.

Reply:
That's great!